crates/beerus-core/tests/beerus.rs

```diff
@@ -3957,6 +3957,37 @@ mod tests {
         assert_eq!(result.unwrap_err().to_string(), expected_error.to_string());
     }
 
+    #[tokio::test]
+    async fn given_empty_result_when_calling_starknet_pending_transactions_then_should_return_error(
```

Comment:
Great that you have been adding tests, are they passing ?

Reply:
They aren't passing because the code doesn't compile from the implementation